server/config/config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  development: {
+    username: 'andeladeveloper',
+    password: null,
+    database: 'docmanager-dev',
+    host: '127.0.0.1',
+    port: 5432,
+    dialect: 'postgres'
+  },
+  test: {
+    username: 'andeladeveloper',
+    password: null,
+    database: 'docmanager-test',
+    host: '127.0.0.1',
+    port: 5432,
+    dialect: 'postgres'
+  },
+  production: {
+    username: 'root',
+    password: null,
+    database: 'database_production',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  }
+};
```

Comment:
It's not advisable to leave your database credentials in your code. I think you can set them in your environment